src/components/molecules/EarthEditor/DatasetPane/DatasetModal/Gdrive/index.tsx

```diff
@@ -0,0 +1,136 @@
+import Text from "@reearth/components/atoms/Text";
+import { metricsSizes } from "@reearth/theme/metrics";
+import AssetCard from "@reearth/components/molecules/Common/AssetModal/AssetCard";
+import AssetListItem from "@reearth/components/molecules/Common/AssetModal/AssetListItem";
+import Button from "@reearth/components/atoms/Button";
+import useHooks, { GoogleSheet, SheetParameter } from "./hooks";
+
+export type Props = {
+  onReturn: () => void;
+  onSheetSelect: (sheet: SheetParameter) => void;
+  syncLoading: boolean;
+};
+
+const Gdrive: React.FC<Props> = ({ onReturn, syncLoading, onSheetSelect }) => {
+  const intl = useIntl();
+  const theme = useTheme();
+
+  const {
+    isLoading,
+    pickedFile,
+    pickedFileSheets,
+    selectedSheet,
+    handleAuthClick,
+    handleSheetSelect,
+  } = useHooks(onSheetSelect);
+
+  return (
+    <>
+      <StyledIcon icon={"arrowLongLeft"} size={24} onClick={onReturn} />
+      {syncLoading && <Loading />}
+      <Flex justify="center" direction="column" align="center">
+        <Flex justify="center" align="center">
+          <GdriveIcon size={32} icon="googleDrive" />
+          <Text size="m" color={theme.main.strongText} weight="bold">
+            {intl.formatMessage({ defaultMessage: "Connect with Google Drive" })}
+          </Text>
+        </Flex>
+        <Divider margin="24px" />
+        {pickedFile?.id && (
+          <>
+            <AssetCard
+              isImage={false}
+              name={pickedFile.name}
+              cardSize="medium"
+              icon="sheetFile"
+              iconSize="50px"
+              onCheck={() => handleAuthClick()}
+            />
+            <Divider margin="24px" />
+            <AssetWrapper direction="column" justify="space-between">
+              <AssetList wrap="nowrap" direction="column" justify="space-between">
+                {pickedFileSheets?.map((sheetItem: GoogleSheet) => (
+                  <AssetListItem
+                    key={sheetItem.properties.sheetId}
+                    asset={{ id: sheetItem.properties.sheetId, name: sheetItem.properties.title }}
+                    isImage={false}
+                    onCheck={() => {
+                      handleSheetSelect({
+                        id: sheetItem.properties.sheetId,
+                        name: sheetItem.properties.title,
+                      });
+                    }}
+                    selected={sheetItem.properties.sheetId === selectedSheet?.id}
+                  />
+                ))}
+              </AssetList>
+            </AssetWrapper>
+            <Divider margin="24px" />
+          </>
+        )}
+
+        {!pickedFile?.id && (
+          <>
+            <Text
+              size="m"
+              color={theme.colors.text.weak}
+              otherProperties={{ marginBottom: metricsSizes["m"] + "px" }}>
+              {intl.formatMessage({
+                defaultMessage: "Re:earth supports Google Sheets, and CSV file to upload.",
```

Comment:
Sorry missed this. Should be "Re:Earth supports uploading Google Sheets and CSV files."